contracts/src/token/psp22/psp22.rs

```diff
@@ -44,7 +55,7 @@ pub use psp22::{
 
 pub const STORAGE_KEY: u32 = openbrush::storage_unique_key!(Data);
 
-#[openbrush::upgradeable_storage(STORAGE_KEY)]
+#[openbrush::storage_item(STORAGE_KEY)]
 #[derive(Default, Debug)]
 pub struct Data {
```

Comment:
this struct is still not upgradeable. it will be stored under a specific storage key, but the mapping key will be generated automatically by ink! anyway